add_annotations_to_table_helper.py

```diff
@@ -16,16 +17,48 @@ def parse_csq_header(vcf_file):
             return match.group(1).split('|')
 
 def parse_csq_entries(csq_entries, csq_fields):
-    transcripts = []
+    transcripts = {}
     for entry in csq_entries:
         values = entry.split('|')
         transcript = {}
         for key, value in zip(csq_fields, values):
             transcript[key] = value
-        transcripts.append(transcript)
+        if transcript['Allele'] not in transcripts.keys():
+            transcripts[transcript['Allele']] = []
+        transcripts[transcript['Allele']].append(transcript)
+    return transcripts
+
+def resolves_alleles(entry, csq_alleles):
```

Comment:
`resolve_alleles` would match the verb forms of the other function names more closely.

Reply:
I changed the method name as suggested.